edison/config.py

```diff
@@ -0,0 +1,36 @@
+import inspect
+import sys
+
+
+def get_config_object(env_keyword: str):
```

Comment:
Is there a reason you are type checking this specific parameter?

Reply:
we decided as a group (Edison group) to use type checking in all our functions, Until now there were no functions so it's seems unnecessary, but, It's here for the consistency and the benefit of more complex functions